portfolio/src/main/webapp/js/scripts.js

```diff
@@ -47,12 +47,12 @@
 async function getData() {
     const response = await fetch('/data');
     const responseJson = await response.json();  
-    const responseLstElem = document.getElementById('greeting-container');
+    const responseLstElem = document.getElementById('comments-container');
```

Comment:
it is usually not a good practice to use abbreviation in the code, as your abbreviation understanding may be different from others

Reply:
okay!